NewForce-Cohort-6/tabloidmvc-the-bit-o-honeys
# Tj ticket17

# Description

PostTagRepository
IPostTagRepository
Tag Repository new TagOnPostIndex



# Testing Instructions
Run NewQuery

INSERT INTO PostTag (PostId, TagId)
VALUES('1', '2')

INSERT INTO PostTag (PostId, TagId)
VALUES('4', '3')

You should be able to see the tags on the "C# is the best language" post, and Archaoptrix



# Checklist:
